### ISSUE TITLE
[OSB] Plan for OSB

### ISSUE THREAD

**Issue by alexdewar** [omicsNLP/Auto-CORPus#23]
- [x] #24 
- [x] #25 
- [x] #26 
- [x] #27 
- [x] #28 
- [x] #29 
- [x] #30 
- [x] #31 
- [x] #32
- [x] #33
- [x] #35 
- [x] #40 
- [x] #41 
- [x] ~Perform a general audit of code, looking at quality and maintainability~
- [x] #74 
- [x] #75 
- [x] #53 
- [x] #62

**Comment by Thomas-Rowlands** [comment]
Unticked pytest due to a problem identified during the hackathon event. This test passes on GitHub during the workflow, as well as locally the first time the repo clones, however it fails subsequently (and correctly).

**Comment by alexdewar** [comment]
I'm not sure why you've unticked it? It's just a todo list and we did it so we checked it off... Did you want to disable pytest or something? Is it not working?

**Comment by Thomas-Rowlands** [comment]
We just want to make sure this isn't closed yet as we want to re-do the pytest script and/or workflow action, since I was about to close it since its all ticked. We could make a separate issue if this is preferred with you guys?

**Comment by Thomas-Rowlands** [comment]
The pytest action has been passing for each build, and it passes once we download/clone the repo and run "poetry run pytest", yet if it is run again it fails. Looking into why it fails, it is correct that it does, however this has meant that builds for some time now have been passing but should not have

**Comment by alexdewar** [comment]
@Thomas-Rowlands Yep, I think opening a new issue is the way to go. We closed the old one by adding the pytest stuff; if it's not working then it's a bug and it would be better to have a new issue describing the problem.

I'm going to close this umbrella issue because all the tasks have been completed.
